### DISCUSSION TITLE
Cipher BF-CBC not supported (Version 0.7.26)

### DISCUSSION THREAD

**Discussion by tr3027** [schwabe/ics-openvpn#1385] [Configuration problems]
After updating to 0.7.26 I cannot connect to VPN anymore. Every attempt to connect ends with the above error. I cannot even see any connection attempt within openvpn log on the server. This is the log I get on phone: [ovpn.log](https://github.com/schwabe/ics-openvpn/files/7303385/ovpn.log)

If I add **--cipher AES-256-GCM** to the manual configuration, I see the connection attempt hits the server, and the connection on phone ends with a VERIFY ERROR because of a *weak CA signature digest*. The root CA is signed with SHA1 unfortunately. I know this is old/weak but cannot replace immediately...

As it used to work until 0.7.26 - is there a possibility to get it running again by some setting or do I have to (temporarily, until the CA can get replaced) downgrade and hold any updates of ics-openvpn? I've the version from Google Play installed.

Thanks for any information!

**Comment by schwabe** [comment]
- Load the legacy provider for BF-CBC, see basic settings in the VPN
- for the weak CA, see the FAQ

**Reply by tr3027** [reply]
Many thanks for your quick response! I don't need the BF-CBC, accepting the weak CA did the job.
 
I read the FAQ prior asking, of course. The *(MD5)* in the FAQ title *Weak (MD5) hashes in ...* did confuse me seemingly. So, thanks for pointing me back again :)
